otter/convergence/steps.py

```diff
@@ -575,6 +580,8 @@ def _rcv3_check_bulk_add(attempted_pairs, result):
         next_step = BulkAddToRCv3(lb_node_pairs=to_retry)
         return next_step.as_effect()
     else:
+        # It's unclear when this condition is reached. Should we really be
+        # returning SUCCESS?
```

Comment:
I think the motivation for this was that if some other error happens, someone on the RCv3 team will manually fix it?  But yeah, wondering if we should return Retry instead.